### ISSUE TITLE
Get drive letters.

### ISSUE THREAD

**Issue by Sven65** [resin-io/drivelist#14]
Hi. I'm using a windows machine and I don't want the

```
[{"device":"\\\\.\\PHYSICALDRIVE0","description":"ST3750528AS ATA Device ","size":"750 GB"},{"device":"\\\\.\\PHYSICALDRIVE1","description":"TOSHIBA MQ01ABD100 ATA  Device","size":"1000 GB"}]
```

Info as this in no way serves of any use to me.
I'd like to see a way of getting the drive letters, for example, ```C:\``` and ```D:\``` instead of the ```PHYSICALDRIVE0``` and ```PHYSICALDRIVE1```.

Thanks

\- Mackan

**Comment by jviotti** [comment]
Hi @Sven65 ,

We luckily have this change implemented in a pull request, and looking forward for people to test it out before merging to `master`.

Do you mind testing https://github.com/resin-io/drivelist/pull/12 and let us know how it goes?

Thanks you very much!

**Comment by Sven65** [comment]
Sure, just tell me how to get it into my environment.

**Comment by jviotti** [comment]
@Sven65 You can run the following npm command to install the specific drivelist branch from GitHub:

```sh
> npm install --save resin-io/drivelist#feature/mountpoint
```

The drive letters will then appear as `mount point` properties.

Let me know if you need any further assistance with the testing, I'll be happy to help.

**Comment by Sven65** [comment]
Okay, when trying to install it, this appears

```
C:\Users\Max\Desktop\node\system>npm install --save resin-io/drivelist#feature/m
ountpoint
npm WARN package.json System@0.0.1 No repository field.
npm WARN package.json System@0.0.1 No README data
npm ERR! addLocal Could not install resin-io/drivelist#feature/mountpoint
npm ERR! Error: ENOENT, stat 'C:\Users\Max\Desktop\node\system\resin-io\drivelis
t#feature\mountpoint'
npm ERR! If you need help, you may report this *entire* log,
npm ERR! including the npm and node versions, at:
npm ERR!     <http://github.com/isaacs/npm/issues>

npm ERR! System Windows_NT 6.1.7601
npm ERR! command "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nod
ejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "--save" "resin-io/drivelist#
feature/mountpoint"
npm ERR! cwd C:\Users\Max\Desktop\node\system
npm ERR! node -v v0.10.25
npm ERR! npm -v 1.3.24
npm ERR! path C:\Users\Max\Desktop\node\system\resin-io\drivelist#feature\mountp
oint
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     C:\Users\Max\Desktop\node\system\npm-debug.log
npm ERR! not ok code 0

```

**Comment by jviotti** [comment]
@Sven65 That's weird. It looks like Windows understand `resin-io/drivelist` as a local file instead of a GitHub repository. It works for me here:

```sh
$ npm install resin-io/drivelist#feature/mountpoint
drivelist@1.2.2 node_modules/drivelist
├── async@1.4.2
├── table-parser@0.0.3
└── lodash@3.10.1
```

What NodeJS and NPM version are you running?

Alternatively, can you try using the `git` url directly?

```sh
> npm install --save git://github.com/resin-io/drivelist.git#feature/mountpoint
```

**Comment by Sven65** [comment]
````
C:\Users\Max\Desktop\node\system>npm -v
1.3.24

C:\Users\Max\Desktop\node\system>node -v
v0.10.25
```

The git url worked..

**Comment by jviotti** [comment]
@Sven65 Great to hear the git url worked. Let us know how it goes!

**Comment by Sven65** [comment]
Works flawlessly!

```json
[{"device":"\\\\.\\PHYSICALDRIVE0","description":"ST3750528AS ATA Device","size":"750 GB","mountpoint":"C:"},{"device":"\\\\.\\PHYSICALDRIVE1","description":"TOSHIBA MQ01ABD100 ATA Device","size":"1000 GB","mountpoint":"D:"}]
```

**Comment by jviotti** [comment]
@Sven65 That's amazing news! Thank you very much your time. I'll publish a new version to NPM and let you know.

**Comment by Sven65** [comment]
No problem, glad to help :P

**Comment by jviotti** [comment]
@Sven65 I've published v1.3.0 to NPM, which includes the `mountpoint` property.

**Comment by Sven65** [comment]
Great! I'll check it out! Thanks
